presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -7889,10 +7890,16 @@ public void testQuantifiedComparisonCornerCases(String query)
         );
         List<QueryTemplate.Parameter> quantifier = new QueryTemplate.Parameter("quantifier").of("ALL", "ANY");
         List<QueryTemplate.Parameter> lhs = new QueryTemplate.Parameter("lhs").of("1", "NULL");
-        List<QueryTemplate.Parameter> operator = new QueryTemplate.Parameter("operator").of("<", ">", "<=", ">=");
+        List<QueryTemplate.Parameter> operator = new QueryTemplate.Parameter("operator").of("=", "!=", "<", ">", "<=", ">=");
         QueryTemplate queryTemplate = new QueryTemplate("SELECT %lhs% %operator% %quantifier% (%subquery%)");
+        ImmutableList<String> excludedInPredicateQueries = ImmutableList.of(
+                "SELECT NULL != ALL (SELECT * FROM (SELECT 1 WHERE false) as empty_table)",
```

Comment:
I find the verbatim query used for exclusion more explicit and thus better telling what exactly needs to be excluded. (For listing a space of test cases, otoh, I think it's better listing possible parameters and saying "use all combinations").

We can address the brittleness by asserting that the excluded query was in the 'all combinations' stream in the first place.